.github/workflows/crowdin-upload.yml

```diff
@@ -0,0 +1,58 @@
+name: Crowdin Upload
+on:
+  workflow_dispatch:
+  schedule:
+    # runs every day at 11:00 AM UTC
+    - cron: "0 11 * * *"
```

Comment:
@nhcarrigan I am thinking this is a bit severe for this project. I would _maybe_ upload once per week. Thoughts?

Reply:
If you're not anticipating a lot of changes to the course, we could definitely scale back the CRON job - or even disable it and only run the upload manually.

Reply:
I say we do it manually.